app/controllers/users_controller.rb

```diff
@@ -1,3 +1,25 @@
 class UsersController < ApplicationController
-  def new; end
+  def new
+    @user = User.new
+  end
+
+  def show
+    @user = User.find params[:id]
+  end
+
+  def create
+    @user = User.new user_params
+    if @user.save
+      flash[:success] = t "static_pages.home.welcome"
+      redirect_to user_url @user
+    else
+      render "new"
```

Comment:
Ưu tiên dùng symbol em nhé
:new

Reply:
render khác redirect_to chỗ nào

Reply:
redirect_to gọi http method điều hướng đến 1 url mới
render hiển thị giao diện mà không điều hướng đến url mới, không reload lại toàn bộ trang